src/drc/api/filters.py

```diff
@@ -13,9 +12,15 @@
 
 
 class EnkelvoudigInformatieObjectListFilter(FilterSet):
+    def zoek_uuids(self, queryset, name, values):
+        values = list(dict(self.data).values())[0]
+        return queryset.filter(uuid__in=values)
+
+    zoek = filters.CharFilter(method="zoek_uuids")
+
     class Meta:
         model = EnkelvoudigInformatieObject
-        fields = ("identificatie", "bronorganisatie")
+        fields = ("identificatie", "bronorganisatie", "zoek")
```

Comment:
De naam van de query parameter zou ik veranderen naar `uuid`, dat is ook in feite waar je op zoekt als gebruiker.